tools/oss-check

```diff
@@ -290,12 +290,14 @@ end
 @repos = [
   Repo.new('Aerial', 'JohnCoates/Aerial'),
   Repo.new('Alamofire', 'Alamofire/Alamofire'),
+  Repo.new('DuckDuckGo', 'duckduckgo/iOS'),
   Repo.new('Firefox', 'mozilla-mobile/firefox-ios'),
   Repo.new('Kickstarter', 'kickstarter/ios-oss'),
   Repo.new('Moya', 'Moya/Moya'),
   Repo.new('Nimble', 'Quick/Nimble'),
   Repo.new('Quick', 'Quick/Quick'),
   Repo.new('Realm', 'realm/realm-cocoa'),
```

Comment:
While we're in here we could update this repo to point to `realm-swift`:

```suggestion
  Repo.new('Realm', 'realm/realm-swift'),
```

I think it was renamed a few years ago.